config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+ELwoAGSNDTiBR6wtiQp5ziavk6ygdBu1gx5YNLDjwIxgBoRWN38C8Tg02HOxgtEDFzSzBGzMcYAyci0r787Ex2RSPHseKdT50P/i4tSzG4/ACyWLF9qUugnLislSiwTcsf5ukdBAz3FstPiYLcQeutRfPrHJAGg7PjOYk5pZFN8jesNewqHas22sYIhh4egQ8vqsG0t4IIctKsZVlJq71ChikgqeJ7j2XtDA/K4k3Vn87kmiIVSIxUX+O2zvwdyGONJLcdW4z30aCaS97wZ7kRqn88gQtl2WJrzau7VRS3Y6oGwxeOp9QdFLvPc+Gw+bd8h16HLrB8LIelKU3yurV/QNYnt5KrjxDNwAJKsW6UW9UX1Xuu2QSustC3bj2Koj4erozNb/NiSGgxBYX/Pkv9r7rTcwO3sIdvhr--G3JfNGDiZe/LzYoB--ZcHEnU6LuZ9f3zEeLfWGng==
```

Comment:
You move this file to the '.gitignore' file. Create 1 empty file 'credentials.yml.enc.example'

Reply:
done

Reply:
done